cluster/kubectl.sh

```diff
@@ -86,6 +86,7 @@ if [[ -z "${KUBECTL_PATH:-}" ]]; then
     "${KUBE_ROOT}/_output/dockerized/bin/${host_os}/${host_arch}/kubectl"
     "${KUBE_ROOT}/_output/local/bin/${host_os}/${host_arch}/kubectl"
     "${KUBE_ROOT}/platforms/${host_os}/${host_arch}/kubectl"
+    "/usr/local/bin/kubectl"
```

Comment:
Does this have to be an absolute path? How about just "kubectl" and use it if it exists on the path